apps/mochi-web/components/explore/index/sections/SummarySection.tsx

```diff
@@ -0,0 +1,170 @@
+                  <div className="font-medium text-xs text-neutral-0 ">
+                    Live
+                  </div>
+                </div>
+                <div className="font-medium text-lg">Current Transactions</div>
+              </div>
+            </div>
+            <div className="font-medium text-5xl text-neutral-900">
+              {loading ? (
+                <Skeleton className="w-64 h-14" />
+              ) : (
+                formatNumber(transactionSummary.current_transactions)
+              )}
+            </div>
+            <div className="flex gap-2 items-center">
+              {loading ? (
+                <Skeleton className="w-24 h-8" />
+              ) : (
+                <>
+                  <div className="font-medium text-xl text-primary-700">
+                    {formatNumber(
+                      transactionSummary.transactions_per_second || 0,
+                    )}
+                  </div>
+                  <div className="text-neutral-600">
+                    transactions per second
+                  </div>
+                </>
+              )}
+            </div>
+          </div>
+          <div className="flex items-center justify-center gap-4 w-full lg:w-auto">
+            {loading ? (
+              <Skeleton className="w-[150px] h-[150px] rounded-full" />
+            ) : (
+              <div className="flex gap-4 items-center">
+                <div className="-m-4">
+                  <PieChart
+                    width={180}
+                    height={180}
+                    margin={{
+                      top: 0,
+                      right: 0,
+                      bottom: 0,
+                      left: 0,
+                    }}
+                  >
+                    <Pie
+                      data={[
+                        {
+                          value: transactionSummary.success_transactions,
+                          fill: '#088752',
+                        },
+                        {
+                          value: transactionSummary.fail_transactions,
+                          fill: '#E02D3C',
+                        },
+                      ]}
+                      dataKey="value"
+                      stroke="none"
+                      startAngle={-270}
+                      endAngle={90}
+                    />
+                  </PieChart>
+                </div>
+                <div className="flex flex-col gap-1">
+                  <div className="flex gap-2 items-center">
+                    <div className="w-2 h-2 rounded-full bg-danger-700" />
+                    <div className="text-sm">Failed</div>
+                  </div>
+                  <div className="flex gap-2 items-center">
+                    <div className="w-2 h-2 rounded-full bg-success-700" />
+                    <div className="text-sm">Success</div>
+                  </div>
```

Comment:
Can we use the color codes from the semantic definitions?

https://github.com/consolelabs/mochi-ui/blob/852f9132e437869a7cbace15f27165ba0ec366f8/packages/theme/src/colors/semantic.ts#L84